redmine/manifests/redmine-svc.yaml

```diff
@@ -9,8 +9,9 @@ metadata:
 spec:
   type: LoadBalancer
   ports:
-  - port: 80
-    targetPort: redmine-http
+  - name: http
```

Comment:
I do think `http` sounds better. I was thinking if we use IANA defined service names, it would make it easier to decide in a general way for future charts.